package.json

```diff
@@ -49,71 +47,54 @@
-    "sass:build": "run-p sass-build-task:scss-compile:*",
-    "presass:copy": "cross-env copyfiles -f src/aurelia-slickgrid/styles/*.png dist/styles/css",
-    "sass:copy": "cross-env copyfiles -f src/aurelia-slickgrid/styles/*.scss dist/styles/sass",
-    "postsass:copy": "cross-env copyfiles -f src/aurelia-slickgrid/styles/*.png dist/styles/sass",
-    "presass:watch:bootstrap": "cross-env npm run sass-build-task:scss-compile:bootstrap",
-    "sass:watch:bootstrap": "cross-env npm run sass-build-task:scss-compile:bootstrap -- -watch",
-    "presass:watch:material": "cross-env npm run sass-build-task:scss-compile:material",
-    "sass:watch:material": "cross-env npm run sass-build-task:scss-compile:material -- -watch",
-    "presass:watch:salesforce": "cross-env npm run sass-build-task:scss-compile:salesforce",
-    "sass:watch:salesforce": "cross-env npm run sass-build-task:scss-compile:salesforce -- -watch",
-    "sass:watch": "run-p sass:watch:bootstrap sass:watch:material sass:watch:salesforce sass:copy:watch",
-    "sass:copy:watch": "nodemon --ext scss --watch src/aurelia-slickgrid/styles/*.scss --exec \"npm run sass:copy\"",
     "start:dev": "webpack-dev-server -d --port=9000 --env.development --config webpack.config.js",
     "start:webpack:dev": "node node_modules/webpack-dev-server/bin/webpack-dev-server -d --port=9000 --env.server --config webpack.config.js",
     "cypress": "cd test/cypress && node node_modules/cypress/bin/cypress open",
     "cypress:ci": "cd test/cypress && npm run cypress:ci",
-    "test:jest": "npx jest --watch --config test/jest.config.js",
-    "test:jest:coverage": "npx jest --runInBand --coverage --config test/jest.config.js",
+    "test:jest": "npx jest --watch --detectOpenHandles --config test/jest.config.js",
+    "test:jest:coverage": "npx jest --detectOpenHandles --runInBand --coverage --config test/jest.config.js",
     "build:demo": "webpack --progress -p --env.production --env.extractCss",
     "serve:demo": "http-server ./docs -p 9000 -a localhost"
   },
   "dependencies": {
+    "@slickgrid-universal/common": "^0.4.0",
     "aurelia-event-aggregator": "^1.0.3",
     "aurelia-framework": "^1.3.1",
-    "aurelia-i18n": "^2.3.2",
+    "aurelia-i18n": "^3.1.4",
     "aurelia-pal": "^1.8.2",
     "aurelia-templating-resources": "^1.13.1",
-    "dompurify": "^2.2.2",
+    "dompurify": "^2.2.3",
     "excel-builder-webpacker": "^2.0.1",
     "flatpickr": "^4.6.6",
     "jquery": "^3.5.1",
     "jquery-ui-dist": "^1.12.1",
     "lodash.isequal": "^4.5.0",
     "moment-mini": "^2.24.0",
+    "multiple-select-modified": "^1.3.8",
     "slickgrid": "^2.4.32",
-    "text-encoding-utf-8": "^1.0.2"
+    "text-encoding-utf-8": "^1.0.2",
+    "typescript": ">=4.1.2"
+  },
+  "peerDependencies": {
+    "i18next": ">= 14.1.1"
   },
   "devDependencies": {
+    "@slickgrid-universal/excel-export": "^0.4.1",
```

Comment:
I've noticed the example uses v `0.5.1`. Besides that why is this listed as a devDependency here but as normal dependency for the consumer -> the samples?

Reply:
I've pushed `0.5.1` yesterday that includes the Composite Editor as a new Lerna package and was waiting to merge this PR and then create another PR to update to `0.5.1` and add Composite Editor. They are all `devDependencies` because they are really optional, I add them here so I can run Cypress tests and make sure everything works. The only Lerna that is essential is the `/common`, all other ones are optional, does that make sense?

Reply:
I added a note in the aurelia-slickgrid-demo saying that those are installed only for demo purposes and should be removed from your dependencies if you don't use the feature, see this [readme note](https://github.com/ghiscoding/aurelia-slickgrid-demos/pull/4/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

Reply:
Ok cool. I was wondering whether that was causing the rjs issue

Reply:
I assume this is resolve then? Let me know if there's anything else to do before merging this PR, probably later tonight, so I can create the next PR for Composite Editor. Also note that I create a `version2` branch last week, so I still have previous code is something goes awfully wrong.

Reply:
Yeah, as said rjs has only limited spread and there is a workaround. So this shouldnt block the release